examples/agent-examples.yaml

```diff
@@ -0,0 +1,76 @@
+# Example: Agents and Agent Relationships
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - alice: https://alice.example/
+    bob: https://bob.example/
+    fablab: https://fablab.example/
+    coop: https://coop.example/
+
+'@graph':
+
+  # Agents
+
+  - '@id': alice:
+    '@type': foaf:Person
+    name: 'Alice'
+    image: 'https://alice.example/avatar.png'
+    url: 'https://alice.example'
+    primaryLocation: 'https://somelocation.example'
+    note: 'Alice is a mechanical engineer who likes to work in the fablab.'
+
+  - '@id': bob:
+    '@type': foaf:Person
+    name: 'Bob'
+
+  - '@id': fablab:
+    '@type': org:Organization
+    name: 'Driftless Fablab'
+
+  - '@id': coop:
+    '@type': org:Organization
+    name: 'Community Tool Lending Coop'
+
+  # Roles
+
+  - '@id': fablab:52f0e212-3c4f-4d27-b345-5e964c135824
+    '@type': AgentRelationshipRole
+    name: 'member'
+    rdfs:label: 'is member of'
+    inverseLabel: 'has member'
+    note: 'Both persons and organizations can be members of this fablab.'
```

Comment:
don't we use `skos:note` ?

Reply:
hmmm - i think this one is defined as vf:note with range of skos:note -- what is the best way to do this kind of thing?

Reply:
> defined as vf:note with range of skos:note

such definition makes no sense, let's just use skos:note directly

Reply:
ok

Reply:
you can cover that in the context file, right?